README.md

```diff
@@ -274,189 +285,506 @@ This is defined in libocpp/include/ocpp/v16/charge_point.hpp and takes the follo
+
+  Notifies the user of libocpp that the Operative/Inoperative state of an EVSE has
+  changed. If as a result the state of connectors changed as well, libocpp will
+  additionally call the connector_effective_operative_status_changed_callback for
+  each connector.
+
+- `get_customer_information_callback`
+
+  Returns human readable customer information based on the CertificateHashDataType,
+  IdToken and Customer Identifier passed in.
+
+- `get_log_request_callback`
+
+  Callback to return logs
+
+- `is_reservation_for_token_callback`
+
+  Check if the current reservation for the given evse id is made for the id
+  token / group id token.
+
+- `is_reset_allowed_callback`
+
+  Callback if reset is allowed. If evse_id has a value, reset only applies
+  to the given evse id. If it has no value, applies to complete charging station.
+
+- `ocpp_messages_callback`
 
+  Callback to congfigure ocpp message logging.
 
+- `pause_charging_callback`
+
+  Used to request pausing of charging, the "connector" parameter instructing which
+  connector/EVSE to pause.
+
+- `remote_start_transaction_callback`
+
+  Called when the request can be accepted. The boolean authorize_remote_start
+  indicates if Authorize.req needs to follow or not
+
+- `reset_callback`
+
+  Performs a reset of the requested type
+
+- `security_event_callback`
+
+  Used to react to a security event callback. This callback is
+  called only if the SecurityEvent occured internally within libocpp.
+  Typically this callback is used to log security events in the security log.
+
+- `set_charging_profiles_callback`
+
+  Indicates when a charging profile is received and accepted.
+
+- `stop_transaction_callback`
+
+  Used to stop a transaction. Called when the idTagInfo.status of a
+  StartTransaction.conf is not Accepted, when a RemoteStopTransaction.req is
+  received, or when an UnlockConnector.req is received.
+
+- `time_sync_callback`
+
+  Called on boot notification if the TimeSource ControllerComponent contains
+  Heartbeat.
+
+- `transaction_event_callback`
+
+  Called when a transaction_event was sent to the CSMS.
+
+- `transaction_event_response_callback`
+
+  Called when a transaction_event_response was received from the CSMS.
+
+- `unlock_connector_callback`
+
+  Used by libocpp to force unlock a connector
+
+- `update_firmware_request_callback`
+
+  Initiates a firmware update request. Triggers a security event notification
+  if the certificate is Invalid or Revoked.
+
+- `validate_network_profile_callback`
+
+  Validates the submitted Network Profile. Is Rejected if
+  - No callback registered to validate network profile
+  - CSMS attempted to set a network profile with a lower securityProfile
+  - CSMS attempted to set a network profile that could not be validated
+  - Network profile could not be written to the device model storage
+
+- `variable_changed_callback`
+
+  Called when a variable has been changed by the CSMS
+
+### Initialize the database
```

Comment:
I think something is off with the header levels here. "Initialize the database" and "Register event callbacks and on_handlers" are both at the same level as "Integrating this library with OCPP 2.0.1", which (at least for the callbacks heading) seems wrong.